### PR TITLE
fix deletion logging and correlation id of notifications

### PR DESCRIPTION
**Pull request by KirilKabakchiev**
* fix logging 
* make notification correlation ID to be SMDB's notification iD

### CODE REVIEW

File: pkg/sbproxy/notifications/producer.go [hunk 1]
```diff
@@ -302,13 +300,24 @@ func (p *Producer) connect(ctx context.Context) error {
...
 			}
 		}
+
 		return err
 	}
+
+	// The websocket connection was successfully established but with unknown revision, therefore trigger resync
+	if p.lastNotificationRevision == unknownRevision {
+		log.C(ctx).Info("Websocket connection established with unknown revision. Triggering resync...")
+		resyncChan <- struct{}{}
```
Comment (dpanayotov) [inline pkg/sbproxy/notifications/producer.go]: connect does multiple things now. Dunno if the readability is so bad now, that we'd rather violate solid

File: pkg/sbproxy/notifications/handlers/broker_handler.go [hunk 2]
```diff
@@ -97,7 +97,7 @@ func (bnh *BrokerResourceNotificationsHandler) OnCreate(ctx context.Context, pay
 
 	existingBroker, err := bnh.BrokerClient.GetBrokerByName(ctx, brokerToCreate.Resource.Name)
 	if err != nil {
-		log.C(ctx).Warnf("Could not find platform broker in platform with name %s: %s", brokerToCreate.Resource.Name, err)
+		log.C(ctx).Debugf("Could not find platform broker in platform with name %s: %s", brokerToCreate.Resource.Name, err)
```
Comment (dotchev) [inline pkg/sbproxy/notifications/handlers/broker_handler.go:100]: this log looks very similar to the one on line 104

File: pkg/sbproxy/notifications/producer.go [hunk 3]
```diff
@@ -159,39 +159,36 @@ func (p *Producer) run(ctx context.Context, messages chan *Message, group *sync.
...
+		if err := p.connect(ctx, resyncChan); err != nil {
+			if err != errLastNotificationGone {
+				// Wait for the reconnect delay to pass before attempting to reconnect
+				log.C(ctx).WithError(err).Error("could not connect websocket")
+				select {
+				case <-ctx.Done():
+					log.C(ctx).Info("Context cancelled. Terminating notifications handler")
+					return
+				case <-time.After(p.producerSettings.ReconnectDelay):
+					log.C(ctx).Debug("Reconnection delay elapsed. Reattempting to establish websocket connection")
```
Comment (dotchev) [inline pkg/sbproxy/notifications/producer.go]: "reestablish" is much more popular than "reattempt", see https://trends.google.com/trends/explore?q=reattempt,reestablish

File: pkg/sbproxy/notifications/producer.go [hunk 4]
```diff
@@ -276,7 +273,7 @@ func (p *Producer) ping(ctx context.Context, done chan<- struct{}) {
 	}
 }
 
-func (p *Producer) connect(ctx context.Context) error {
+func (p *Producer) connect(ctx context.Context, resyncChan chan struct{}) error {
```
Comment (dotchev) [inline pkg/sbproxy/notifications/producer.go]: why mix ws connect with resync?

**Review by dotchev — DISMISSED** [review]

**Review by dpanayotov — APPROVED** [review]

File: pkg/sbproxy/notifications/producer.go (hunk 4 above)
Comment (KirilKabakchiev) [reply pkg/sbproxy/notifications/producer.go]: I wanted to remove the shouldResync variable but for now this is reverted

**Pull request merged**